go/vt/vtexplain/testdata/multi-output/selectsharded-output.txt

```diff
@@ -166,3 +166,9 @@ select id, 'abc' as test from user where id = 1 union all select id, 'def' as te
 1 ks_sharded/-40: select id, 'abc' as test from `user` where id = 1 union all select id, 'def' as test from `user` where id = 1 union all select id, 'ghi' as test from `user` where id = 1 limit 10001 /* union all */
 
 ----------------------------------------------------------------------
+select id from user where not id in (select col from music where music.user_id = 42) and id in (select col from music where music.user_id = 411)
+
+1 ks_sharded/40-80: select col from music where music.user_id = 42 limit 10001
+2 ks_sharded/40-80: select col from music where music.user_id = 411 limit 10001
+
+----------------------------------------------------------------------
```

Comment:
this does not tell what happened to the main query, where it is executed.
I have less knowledge here. so cc: @aquarapid

Reply:
It's not ideal, but acceptable;  maybe make a note as a cleanup/enhancement item to add visibility of the top-level query?